cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/WidgetIndex.kt

```diff
@@ -28,6 +31,9 @@ class WidgetIndex {
             BambooDeploymentWidget::class.java.simpleName -> BambooDeploymentWidget(vertx, config)
             BambooPlanWidget::class.java.simpleName -> BambooPlanWidget(vertx, config)
             AemHealthcheckWidget::class.java.simpleName -> AemHealthcheckWidget(vertx, config)
+            TextWidget::class.java.simpleName -> TextWidget(vertx, config)
+            IframeEmbedWidget::class.java.simpleName -> IframeEmbedWidget(vertx, config)
+            WorldClockWidget::class.java.simpleName -> WorldClockWidget(vertx, config)
```

Comment:
Or i can create only one new widget type like `CopyToContentWidget` that will have third argument with set of props to copy. WDYT ?

Reply:
I'd leave as-is. By changing to what you describe, these props would have to be defined somewhere, and we would be essentially talking about placing the props as a class, or as a variable somewhere.